main.go

```diff
@@ -106,6 +108,32 @@ func main() {
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
+	// Parse environment variable options, prefixed with "VSO_"
+	if err := vsoEnvOptions.Parse(); err != nil {
+		os.Stderr.WriteString(fmt.Sprintf("Failed to process environment variable options: %q\n", err))
+		os.Exit(1)
+	}
+
+	// Set options from env if any are set
+	if vsoEnvOptions.OutputFormat != "" {
```

Comment:
Should we document the env vars in the corresponding flags' usage?

Reply:
Sure that makes sense. Added in e066367.